dio/pubspec.yaml

```diff
@@ -18,4 +18,4 @@ dev_dependencies:
   test: ^1.21.1
   coverage: ^1.3.0
   mockito: ^5.1.0
-  http: ^0.13.4
+  http: ^0.13.0
```

Comment:
@ueman do you remember if we used min `.4` for some reason or is it fine to align with the `sentry` dep which is `.0`?

Reply:
The http dep shouldn't be needed in the dio package at all. After all, dio is a replacement for http.

Reply:
Oh, it's a dev dependency in which case it shouldn't matter, since it doesn't affect users. Still, I wonder why it's there

Reply:
will check if its used in tests, otherwise will remove it, it'd be nice to have a linter that warn us something isn't being used :D

Reply:
yeah it was a mistake, removing it.